src/main/java/frc/robot/util/StoppingCounter.java

```diff
@@ -14,14 +14,17 @@ public StoppingCounter(String name, int targetCounts) {
 	}
 	
 	public boolean isFinished(boolean conditionToCheck) {
-		if ( conditionToCheck) {
+		if (conditionToCheck) {
 			currentCounts++;
 		}
 		else {
 			currentCounts = 0;
 		}
 		SmartDashboard.putNumber("StopCounter::" + name, currentCounts);
 		return currentCounts > targetCounts;
+	}
 
+	public void reset() {
+		this.currentCounts = 0;
 	}
 }
```

Comment:
You need to add an initialize() method and zero your internal parameters in that method. 
I have a template installer (InstallEntechVSCodeMods.bat) that let's you use the right click New Class/Command menu.